xray/test/test_dataarray.py

```diff
@@ -457,16 +456,6 @@ def test_isel_points(self):
         actual = da.isel_points(y=[1, 2], x=[1, 2], dim=['A', 'B'])
         assert 'points' in actual.coords
 
-    def test_isel_points(self):
```

Comment:
Yes, we had a duplicate test in there (`test_isel_points` exists above and is much more thorough).